pkg/receiver/gitproviderreceiver/README.md

```diff
@@ -9,32 +9,40 @@
 
 [development]: https://github.com/open-telemetry/opentelemetry-collector#development
 [liatrio]: https://github.com/liatrio/liatrio-otel-collector
-<!-- end autogenerated section --> 
+<!-- end autogenerated section -->
 
 The Git Provider receiver scrapes data from Git vendors.
 
-As a starting point, this receiver can infer many of the same core git
-metrics across vendors, while being able to receive additional data specific to
-vendors. 
+As a starting point, this receiver can infer many of the same core git metrics across vendors, while being able to receive additional data specific to vendors.
 
-The current default set of metrics common across all vendors can be found in
-[documentation.md](./documentation.md).
+The current default set of metrics common across all vendors can be found in [documentation.md](./documentation.md).
 
-These default metrics can be used as leading indicators to the DORA metrics; helping 
-provide insight into modern-day engineering practices.
+These default metrics can be used as leading indicators to the DORA metrics; helping provide insight into modern-day engineering practices.
+
+## GitHub Metrics
 
 The current metrics available via scraping from GitHub are:
-* repository branch count
-* repository branch time 
-* repository count
-* repository branch count
-* repository contributor count
-* repository pull request time
+
+- Repository branch count
```

Comment:
any particular reason for all these little changes?

Reply:
Just a linter thing in my IDE, I've always preferred `-` over `*` bullet points and just instinctively change them. I can revert it back so we have a more consistent style.

Reply:
naw it's fine, just asking

Reply:
probably would be good to have a linter config for this so it's consistent across the board

Reply:
How about [markdownlint][0]?

It has support for a CLI, GitHub Actions, VSCode, Vim, Neovim, and more.

[0]: https://github.com/DavidAnson/markdownlint

Reply:
sounds good to me

Reply:
I added Markdownlint as a pre-commit config so we shouldn't have to worry about this going forward.